src/main/org/nlogo/api/VersionHistory.scala

```diff
@@ -195,6 +195,16 @@ object VersionHistory {
     remove3d(version) == "NetLogo 4.2pre3" ||
     remove3d(version) == "NetLogo 4.2pre4"
 
+  def olderThan52(version: String) =
```

Comment:
This doesn't seem fully correct to me. Suppose NetLogo 5.0.6 and/or NetLogo 5.1.1 were to be released.